build.gradle.kts

```diff
@@ -73,9 +73,27 @@ allprojects {
     }
 }
 
+val buildTask = tasks.register("buildNonMkdocs")
+
 subprojects {
     plugins.apply("release-dependencies-diff-create")
 
+    // Allows avoiding building these modules as part of CI as they are also built for mkdocs.
+    if (!path.startsWith(":demos:mkdocs:")) {
```

Comment:
I believe this covers the 4 types of modules that we have. if you can think of any others, please let me know.